cmd/paratime/show.go

```diff
@@ -26,8 +26,8 @@ const (
 	blockLatest = "latest"
 )
 
-var blockCmd = &cobra.Command{
-	Use:   "block <number> [ <tx-index> | <tx-hash> ]",
+var showCmd = &cobra.Command{
```

Comment:
So the idea is then to duplicate this thing for consensus blocks? But there is already a `oasis network show` which does something completely different? Will we need `oasis network show block ...` and then (for consistency) `oasis paratime show block ...`?

Reply:
Actually I would merge all of those. So `oasis paratime show 12345` would show you the block round `12345`. If there will be some ambiguity, we could do `oasis paratime show block:12345` or `oasis paratime show paratime:12345`. Same for the `network`.

Reply:
Ok.